soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1052

```diff
@@ -44,6 +44,13 @@ config I2C_MCUX_LPI2C
 
 endif # I2C
 
+if USB
+
+config USB_DC_NXP_EHCI
+	default y
```

Comment:
This can move up a level to `soc/arm/nxp_imx/rt/Kconfig.defconfig.series`. See commit 501db1e0b96133cd5377c518e681e8f82114f836

Reply:
@MaureenHelm thanks, I have updated it.